src/scrapers/base-scraper-with-browser.js

```diff
@@ -9,7 +9,7 @@ const VIEWPORT_WIDTH = 1024;
 const VIEWPORT_HEIGHT = 768;
 
 function getKeyByValue(object, value) {
-  return Object.keys(object).find(key => object[key] === value);
```

Comment:
@chilikla I would rather use `includes(...)` as it's the modern way of doing `indexOf(...) > -1`

Reply:
I'm not sure your use case is possible, but maybe I didn't understand it.

As far as I understand, if `object[key]` (the success URL, as a string and not an array) equals `https://some.url.com/account`, and `value` (current URL) equals `https://some.url.com/account/error`, `object[key].includes(value)` (your suggestion is of course better) will return false. And this is still true if `object[key]` is an array.

If you agree with me, I will just commit the change to `object[key].includes(value)` and we are done.

Reply:
@chilikla AFAIK, `array.includes(value)` returns true only if the value EQUALS an item in the array.
so `['apple', 'orange'].includes('app')` will return false, whereas `'apple'.includes('app')` will return true.